src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersion.cs

```diff
@@ -40,5 +40,6 @@ public enum SchemaVersion
         V28 = 28,
         V29 = 29,
         V30 = 30,
+        V31 = 31,
```

Comment:
Is there a SchemaConstants.Latest variable somewhere too?

Reply:
Yes, I missed the max value in SchemaVersionConstants. Updated.